common-java-lib-package/src/main/resources/bin/configure.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2019, 2020
+################################################################################
+
+chmod a+x libzowe-attls.so
+extattr +p libzowe-attls.so
```

Comment:
This file will be executed from component root directory, so we need to prefix with `bin/` to find the .so files correctly.

```
chmod a+x bin/libzowe-attls.so
extattr +p bin/libzowe-attls.so
```

Reply:
I've updated paths. Thank you!